nautobot/ipam/factory.py

```diff
@@ -377,9 +377,7 @@ def children(self, create, extracted, **kwargs):
 
         # Default to maximum of 4 children unless overridden in kwargs
         max_count = int(kwargs.pop("max_count", 4))
-        child_count = faker.Faker().pyint(min_value=0, max_value=min(max_count, self.prefix.size))
-        if child_count == 0:
-            return
+        child_count = faker.Faker().pyint(min_value=1, max_value=min(max_count, self.prefix.size))
```

Comment:
This fixed the current `next` test failures. We weren't creating any IP Addresses for IPv6 because we had multiple random chances to not create children. This `random.pyint` was redundant after the above (line 361) `random.choice` so it should be fine to change.

Reply:
Do we ever have a case where `min(max_count, self.prefix.size)` is 0 or is it always `>= 1`?

Reply:
That's a good point. If someone explicitly requests `children__max_count=0`, this could break. I'll add a guard rail for that case. Otherwise, `self.prefix.size` should always be at least 1.